api/src/test/nfl-teams.test.ts

```diff
@@ -15,13 +15,31 @@ const app = new App([nflTeamsRoute]);
 const nflteams = nflTeamsRoute.nflteam.nflteam;
 const mockNFLTeamsList : NFLTeam[] = mockNFLTeams; 
 
+beforeAll(async () => {
+  await DB.NFLTeams.truncate();
+});
 
 afterAll(async () => {
   await new Promise<void>(resolve => setTimeout(() => resolve(), 500));
 });
 
 describe('Testing NFL Teams', () => {
 
+  describe('[POST] /nfl-teams', () => {
```

Comment:
I thought about doing this but didnt because I would have to call the service directly because the route does not actually take parameters it just hits another external endpoint

Reply:
ah ok, sounds good